nimbus.fml.yaml

```diff
@@ -96,12 +81,12 @@ features:
         description: >
           If true, we show the Upgrade onboarding screen when the user upgrades the version.
         type: Boolean
-        default: false
+        default: true
```

Comment:
We want to keep showing the Upgrade screen? after the experiment is finished?

Reply:
Uh, so the values here don't matter because I've created a separate [story](https://mozilla-hub.atlassian.net/browse/FXIOS-5399) to get the actual values for 109.

I changed these to reflect what we had set as default for prod in the experiment. I can change them back if you think that's a better idea, though.